.gitattributes

```diff
@@ -1 +1,2 @@
 CHANGELOG.md merge=union
+src/__generated__/*.graphql.ts linguist-generated
```

Comment:
Good stuff, thanks @pchaigno 🥇

Reply:
@orta - can you explain whats going on here?

Reply:
I can - here's the historical context: https://twitter.com/pchaigno/status/961134607113637889

Reply:
tyty, makes sense 👍